Wallet Provider Onboarding

MartinRobomaze opened the pull request:
This pull request adds a description of the onboarding process of Wallet Providers and their Wallet Solutions to the Wallet Provider Trusted list (issue #28).

MartinRobomaze committed:
initial draft of wallet onboarding process

lj-raidiam commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+    - Beneficiaries and Associated Partners providing a prototype of a EUDI Wallet for natural persons
+    - Beneficiaries and Associated Partners providing a prototype of a European Business Wallet
+- **Secondary Actors [WEBUILD MVP]**:
+    - Ecosystem Authority: WEBUILD WP4 Trust Infrastructure group
```
We need to make it plural as we may or are are going to have couple Registrars and Registers. The trust infra WG will full the role of LOTL provider, while Beneficiaries and APs who provide those registers shall act as Authorities true member states. A wallet provider can register in any of these registers.

lj-raidiam commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+    - The onboarding process for wallet providers is formally defined and documented in a harmonized manner that aligns
+      with EU regulatory and technical frameworks.
+    - Pilot implementations (MVP) successfully demonstrate wallet providers' onboarding
+    - All wallet providers within WEBUILD are included on a publicly accessible trusted list maintained by the WP4 Trust
```
As above. We will have couple Registrars and Registers to mimic couple member states so WP4 will rather have LOTL and couple TLs.

LeoneRiello74 commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+
+## Terminology
+
+(1) ‘wallet solution’ means a combination of soft wware, hardware, services, settings, and configurations, including
```
software

LeoneRiello74 reviewed (approved):
for me it's fine. I would not include chapter three as verification of WUA, because it's in scope of pid issuance not of wallet provider onboarding, and the verification of the chain of the certificate against the TL is standard....

lj-raidiam commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+    - Beneficiaries and Associated Partners providing a prototype of a EUDI Wallet for natural persons
+    - Beneficiaries and Associated Partners providing a prototype of a European Business Wallet
+- **Secondary Actors [WEBUILD MVP]**:
+    - Ecosystem Authority: WEBUILD WP4 Trust Infrastructure group
```
If we are effectively mimicking a member state having a pair of TLP + WRP Registry, how will we manage mimicking multiple Registrars and Registries per state but just a single TLP? 

I would rather say that WP4 is at the LOTL level, while we have APs+Beneficiaries that provide TLPs and/or Registries are authorities.

lj-raidiam replied:
@cklugow, we discussed this today in the meeting and agreed on starting with the single trusted list for wallet providers. I would like to suggest certain rewording to a bit more explicitly indicate that while a single trusted list is a baseline that enables testing, our aspiration would be to have LOTL for wallet providers. At least if time permits. Please have a look at the suggestion above. 

I mentioned WRPs in this context, as mimicking an ecosystem with more than a single member state is going to require more than one registry, and more than one TLP, which implies having LOTL anyways. Taking it into account, having a single TL for WP doesn't seem to simplify a lot.

peppelinux replied:
please reevaluate our assumptions according to the latest updates in https://github.com/webuild-consortium/wp4-trust-group/pull/23 and any further review from you

cklugow committed:
Update task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md

Co-authored-by: Lukasz Jaromin <lukasz.jaromin@raidiam.com>

cklugow commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+
+- **Primary Actor [WEBUILD MVP]**:
+    - Beneficiaries and Associated Partners providing a prototype of a EUDI Wallet for natural persons
+    - Beneficiaries and Associated Partners providing a prototype of a European Business Wallet
```
Thank you for your feedback, @lj-raidiam. We would leave it up to the wallet providers whether they provide an EUDI wallet or a Business wallet. As long as they pass the wallet conformity test in WEBUILD we would list their solution.

cklugow committed:
Updated Title in Line 6

MartinRobomaze committed:
fixed typos etc

IDUhammer committed:
Update README.md

IDUhammer committed:
Update README.md

IDUhammer commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,391 @@
+[WEBUILD MVP]
+
+- There will be 1 Trusted List for all wallet providers in WEBUILD to reduce complexity. WP4 Trust Infrastructure group
+  may later decide to set up several Trusted Lists that are included in 1 List of Trusted Lists.
```
The changes reflect the discussion in the meeting and have been adjusted accordingly.

cklugow reviewed (approved):
ETSI EN 319 412 6 moved from "draft" to "standards"

cklugow committed:
date changed

cklugow committed:
MVP/MVP+ labeling an changing the order

cklugow committed:
RACI Matrix filled - first version

cklugow committed:
added link to Interoperability Test Bed and responsibility of WP4 Testing

cklugow committed:
erased obsolete content (see line 84)

cklugow commented on task1-use-cases/subtask1-3-wallet-provider-onboarding/README.md:
```diff
@@ -0,0 +1,393 @@
+
+## Actors
+
+- **Primary Actor [Extended Scope]**:
```
done here: https://github.com/webuild-consortium/wp4-trust-group/pull/43/commits/b812eeb73f3b2a6e7149912266c7bfcea7672fc1

cklugow committed:
entering an assumption for the service unique identifier of the wallet solution

cklugow committed:
added some references to Regulation (EU) No. 2024/1183 5d

cklugow committed:
changing format

cklugow committed:
changes in format

cklugow committed:
Merge branch 'main' into wallet-provider-onboarding

peppelinux committed:
Merge branch 'main' into wallet-provider-onboarding

peppelinux committed:
fix: removed duplication in wallet provider merge

cklugow committed:
Update README.md

Final check before merging
- no dependency to conformity tests in webuild
- no WUA specification
- editorial changes
